cycode/cli/code_scanner.py

```diff
@@ -336,7 +337,19 @@ def scan_disk_files(context: click.Context, path: str, files_to_scan: List[str])
         if not content:
             continue
 
-        documents.append(Document(file, content, is_git_diff))
+        file_name = file
```

Comment:
we have "perform_pre_scan_documents_actions" function that adds additional files for SCA. I guess it's the best place to add specific logic for IaC. pls check